src/storage/statistics_cl.c

```diff
@@ -342,152 +340,148 @@ stats_dump (const char *class_name_p, FILE * file_p)
       attr_stats_p = &(class_stats_p->attr_stats[i]);
 
       name_p = sm_get_att_name (class_mop, attr_stats_p->id);
-      fprintf (file_p, " Attribute: %s\n", (name_p ? name_p : "not found"));
-      fprintf (file_p, "    id: %d\n", attr_stats_p->id);
-      /*
-         fprintf (file_p, "    ndv: %ld\n", attr_stats_p->ndv);
-       */
-      fprintf (file_p, "    Type: ");
+      fprintf (file_p, " Attribute: %s (", (name_p ? name_p : "not found"));
 
       switch (attr_stats_p->type)
```

Comment:
Instead of a switch statement, what about the following?
```
fprintf (file_p, "%s)\n", pr_type_name (attr_stats_p->type));
```
AS-IS will break consistency when new types are added.
For example, there is no DB_TYPE_JSON in the switch statement.

Reply:
Good point. I fixed it.